doc/installing.rst

```diff
@@ -25,6 +25,20 @@ KWallet, so that the sync client can login automatically.
 You will also find links to source code archives and older versions on the 
 download page.
 
+System Requirements (Version 2.3.3)
```

Comment:
Maybe "Supported Platforms"  instead of "system requirements"?
Because it is still possible to compile from source for many other distro.

Also, if this is about 2.3.3, it should go to the 2.3 branch.  master contains is the documentation for 2.4 now.  but if the platform are the same maybe  adding 2.4 is ok

Reply:
> it is still possible to compile from source for many other distro.

good point.

Wasn't aware of the process. I other repos we first merge to master, before backport to other branches.

Reply:
> Wasn't aware of the process. I other repos we first merge to master, before backport to other branches.

I think @PVince81 @DeepDiver1975 wanted to change that, but never got around it..

Reply:
> (Version 2.3.3)

Please don't mention the version. It should be only in VERSION.cmake, everything else is implicit from the branch/tgz/whatever that contains the README :)

Reply:
@settermjd Can I Haz Variables in Docs? (from VERSION.cmake for example?)

Reply:
@settermjd discussed before in https://github.com/owncloud/client/pull/5762#issuecomment-300486352

Reply:
@michaelstingl Can you just remove the 2.3.3 above then we can merge this?

Reply:
done